rewriter/AttrReader.cc

```diff
@@ -198,7 +198,9 @@ vector<ast::ExpressionPtr> AttrReader::run(core::MutableContext ctx, ast::Send *
 
             ast::MethodDef::Flags flags;
             flags.isAttrBestEffortUIOnly = true;
-            auto reader = ast::MK::SyntheticMethod0(loc, loc, name, ast::MK::Instance(argLoc, varName), flags);
+            // CR cwong: Investigate whether lying about this being synthesized causes other
+            // problems
```

Comment:
Did you want to do that before landing?

(what does `CR` stand for here?)

Reply:
Actually: I just checked, and I think that the only thing this likely affects is the `prop` DSL, so we can deal with that when we get there.